.travis.yml

```diff
@@ -56,14 +56,13 @@ script:
   - export TEST_EXTRA=""
   - if [[ "$COVERAGE" == "true" ]]; then
       export TEST_EXTRA="--cov-config .coveragerc --cov nengo";
-    fi
-  - if [[ "$STATIC" == "false" ]]; then
+    elif [[ "$STATIC" == "true" ]]; then
```

Comment:
It is true that STATIC and COVERAGE won't both be true, but that's not what's happening in this section...

In `install`, we install certain things by default, then different things in the COVERAGE and STATIC cases.

In `script`, we run tests by default, but not in the STATIC case (we run the static analyzers). In the COVERAGE case, we add a few arguments to the tests run by default. So, either the `py.test` invocation has to be added to the COVERAGE case (which feels like unnecessary duplication to me), or keep the logic separate.